storage/raft.go

```diff
@@ -48,68 +48,59 @@ type raftLogger struct {
 	group uint64
 }
 
-func (r *raftLogger) prependContext(format string, v []interface{}) string {
-	var s string
+func (r *raftLogger) context() string {
```

Comment:
Add a comment that bad things will happen if this method returns a string that may contain `%` characters.